libs/common/src/lib/features/features-modal.component.html

```diff
@@ -1,19 +1,22 @@
-<pa-modal-advanced>
+<pa-modal-advanced [fitContentHeight]="!!features">
   <pa-modal-title>{{ 'features.customize_features' | translate }}</pa-modal-title>
   <pa-modal-content>
-    <ng-container *ngIf="features">
-      <div *ngFor="let feature of features | keyvalue">
+    @if (features; as features) {
+      @for (feature of features | keyvalue; track feature) {
         <pa-checkbox [(value)]="features[feature.key]">{{ feature.key }}</pa-checkbox>
```

Comment:
When enabling a feature, and then opening the modal again, it seems this checkbox is unchecked. Is this expected ?

Reply:
did you hit save? when I tried, hitting save was reloading the page and the checkbox was enabled. then clicking on reset put it back as it was (all checkbox disabled)

Reply:
fixed